server_admin/topics/users/proc-enabling-recaptcha-support.adoc

```diff
@@ -2,35 +2,38 @@
 //
 // server_admin/topics/users.adoc
 
-[id="proc-enabling-recaptcha-support_{context}"]
-= Enabling reCAPTCHA Support
+[id="proc-enabling-recaptcha_{context}"]
+= Enabling reCAPTCHA
 
 [role="_abstract"]
 To safeguard registration against bots, {project_name} has integration with Google reCAPTCHA.
 
 Once reCAPTCHA is enabled, you can edit `register.ftl` in your login theme to configure the placement and styling of the reCAPTCHA button on the registration page. 
 
 .Procedure
-. Navigate to the link:https://developers.google.com/recaptcha/[Google Recaptcha website].
+. Enter the following URL in a browser:
+----
+https://developers.google.com/recaptcha/
+----
 . Create an API key to get your reCAPTCHA site key and secret. Note the reCAPTCHA site key and secret for future use in this procedure.
 +
 NOTE: The localhost works by default. You do not have to specify a domain.
 +
 . Navigate to the {project_name} admin console.
-. Click *Authentication* in the left menu. 
+. Click *Authentication* in the main menu. 
 . Click the *Flows* tab.  
 . Select *Registration* from the drop down menu.
-. Set the *reCAPTCHA* requirement to *Required* by clicking the appropriate radio button. This enables
+. Set the *reCAPTCHA* requirement to *Required*. This enables
 reCAPTCHA.  
 . Click *Actions* to the right of the reCAPTCHA flow entry.
 . Click the *Config* link to redirect to the config page.
 . Enter the reCAPTCHA site key generated from the Google reCAPTCHA website, on the config page. 
```

Comment:
I think you can delete ", on the config page" since you are already there.
However, this step is where I got lost in trying the procedure.  I found the screen shot in the existing documentation helped a lot.  Do you feel it's a problem to keep the screen shot?

Reply:
Would you mind adding the images after I make the corrections to this PR?

Reply:
As in this PR is merged and then you add the images as you see fit? I actually do not think that the images are needed and if they are maybe we can refine the text so that it is clearer (and the screenshot is not needed). WDYT? You hold the keys :)